root/about/faq.html

```diff
@@ -151,6 +151,42 @@
 The PAUSE account information need to be updated manually via
 [Edit Account Info](https://pause.perl.org/pause/authenquery?ACTION=edit_cred).
 
+## How to adopt a distribution?
+
+### What does the "Take me over?", "Why not adopt me?", "Looking for help!" mean?
+
+CPAN Distributions can be marked by their authors, or by the CPAN admins, in case the author is not
+available to be "up for adoption". This can be done by adding either one of the following PAUSE accounts
+as co-maintainers: ADOPTME, HANDOFF, or NEEDHELP.
+
+
+### How to become the maintainer or co-maintainer of a CPAN distribution?
+
+* TBD.
```

Comment:
This section could include a link to https://neilb.org/adoption/

Reply:
I think the "TBD" is the only thing left to clarify here.

Reply:
Is there a blog post explaining I could link to?

Reply:
How about http://neilb.org/2013/07/24/adopt-a-module.html ?